src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -2369,6 +2369,7 @@ public RevListCommand all(boolean all) {
                 this.all = all;
                 return this;
             }
+            
```

Comment:
I know this is a silly request, but could you remove the trailing white space from that line?

There are so many formatting issues in the git client plugin and the git plugin that I'd like to avoid adding one more.